lib/aggregate.js

```diff
@@ -397,6 +397,17 @@ Aggregate.prototype.project = function(arg) {
  */
 
 Aggregate.prototype.near = function(arg) {
+  if (arg == null) {
+    throw new MongooseError('Aggregate `near()` must be called with non-nullish argument');
+  }
+  if (arg.near == null) {
```

Comment:
Maybe while you are at, document those properties in the jsdoc, or is the link to the mongodb docs enough?

Reply:
I added a note in jsdoc, since Mongoose is doing validation we should explain the validation step in the jsdoc.